src/switch_rtp.c

```diff
@@ -1175,7 +1180,7 @@ static void handle_ice(switch_rtp_t *rtp_session, switch_rtp_ice_t *ice, void *d
 								return;
 							}
 
-							if ((rtp_session->rtp_bugs & RTP_BUG_ALWAYS_AUTO_ADJUST)) {
+							if ((!zstr(rtp_auto_adjust_always) && switch_true(rtp_auto_adjust_always)) || (rtp_session->rtp_bugs & RTP_BUG_ALWAYS_AUTO_ADJUST)) {
```

Comment:
zstr is redundant here since switch_true should already validate null string for rtp_auto_adjust_always

Reply:
that is true but it is ambiguous.  zstr make the intent clear.